src/overmap.cpp

```diff
@@ -713,52 +712,52 @@ bool overmap::has_note(int const x, int const y, int const z) const
         return false;
     }
 
-    for (auto &i : layer[z + OVERMAP_DEPTH].notes) {
```

Comment:
The logic is equivalent, and merely switches to an idiom with no benefits in this case.